local-cli/runAndroid/runAndroid.js

```diff
@@ -262,7 +262,7 @@ function runOnAllDevices(args, cmd, packageNameWithSuffix, packageName, adbPath)
     }
 }
 
-function startServerInNewWindow() {
+function startServerInNewWindow(port = 8081) {
```

Comment:
Do we need this default arg? Let's try to only specify the default value once in the command config.